src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+export const MovieCard: React.FC<Movie> = (Props) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = Props;
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
  const {
    title,
    description,
    imgUrl,
    imdbUrl,
  } = movie;
```
Better get movie as a prop and take all the needed props of it using destructuring